www/src/components/LeaveFeedback.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Nav } from '~paragon-react';
+import { useLocation } from '@reach/router';
 
 function LeaveFeedback(props) {
-  const CURRENT_URL = typeof window !== 'undefined' ? window.location.pathname : '';
-  const FEEDBACK_URL = `https://github.com/openedx/paragon/issues/new?title=%5Bparagon-openedx.netlify.app%5D%20Feedback%20(on%20${CURRENT_URL})&amp;labels=docs&template=feedback_template.md`;
+  const CURRENT_URL = useLocation();
```

Comment:
I think here it is not a constant variable anymore. It means we can rename it to `const location = useLocation()`.

Reply:
Thanks, it's make sense 👍